addons/account/models/account_payment.py

```diff
@@ -535,6 +535,8 @@ def create(self, vals_list):
         write_off_line_vals_list = []
 
         for vals in vals_list:
+            # Compute the name in the move
+            vals.pop('name', None)
```

Comment:
This is so weird...

Also, improve your comment

Reply:
I'm pretty sure you did something in `_compute_name` checking the `name` is still relevant regarding what actually change in the dependencies to prevent this issue.